liquibase-dist/pom.xml

```diff
@@ -31,7 +31,7 @@
 	</repositories>
 
 	<properties>
-		<deploy.repositoryId>liquibase</deploy.repositoryId>
+		<deploy.repositoryId>github</deploy.repositoryId>
```

Comment:
This should not be changed.

Reply:
Thanks.  I obviously didn't know what I was doing here.